compiler/lsp-tests/src/Main.hs

```diff
@@ -605,3 +606,83 @@ stressTests run _runScenarios = testGroup "Stress tests"
     makeModule :: String -> [T.Text] -> Session TextDocumentIdentifier
     makeModule name lines = openDoc' (name ++ ".daml") damlId $
         moduleContent name lines
+
+regressionTests
+    :: (Session () -> IO ())
+    -> (Session () -> IO ())
+    -> TestTree
+regressionTests run _runScenarios = testGroup "regression"
+  [ testCase "completion on stale file" $ run $ do
+        -- This used to produce "cannot continue after interface file error"
+        -- since we used a function from GHCi in ghcide.
+        foo <- openDoc' "Foo.daml" damlId $ T.unlines
+            [ "{-# OPTIONS_GHC -Wall #-}"
+            , "daml 1.2"
+            , "module Foo where"
+            , "import DA.List"
+            , ""
+            ]
+        expectDiagnostics [("Foo.daml", [(DsWarning, (3,0), "redundant")])]
+        completions <- getCompletions foo (Position 3 1)
+        liftIO $ completions @?= map mkCompletion
+            [ "Control.Exception.Base"
```

Comment:
Is it necessary to compare against an exact list of expected completions in this test, or sufficient to check that, say `DA.List`, is present? With the full list this test will need to be changed if the default set of visible modules is changed.

Reply:
good point, simplified